UIK-3982/prop-to-remove-last-border (#2438)
semrush/intergalactic: semcore/data-table/CHANGELOG.md

CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).

## [16.3.0] - 2025-09-12

### Added

> - `variant` property. Defines visual variant that adapts the table styling to different usage contexts.

Reviewer: ```suggestion
- `variant` property. Adapts the table styling to different usage contexts, such as in a `Card`.
```
I suggest mentioning Card so that devs can quickly understand what's it about when seeing the changelog

Author: good point, agree :)
